networks/canarynet-uat/peers.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "url": "http://localhost:8080/",
+        "name": "localhost.eth",
+        "address": "0xd5e099c71b797516c10ed0f0d895f429c2781142",
+        "packages": [
+            {
+              "name": "retail",
+              "margin": [5, 100],
+              "safety": "5.5"
+            },
+            {
+              "name": "prime",
+              "margin": [100, 1000],
+              "safety": "3.5"
+            }
+          ]
+    },
+    {
+        "url": "https://api.kayen.io/",
```

Comment:
This we will know only after few days, so even next 2 comments can omit for now

Reply:
https://kayen.v4.uat.opendax.app/ maybe?

Reply:
http://clearport.v4.uat.opendax.app:30051